etc/profile-m-z/qt6ct.profile

```diff
@@ -0,0 +1,69 @@
+# Firejail profile for qt6ct
+# Description: Qt6 Configuration Utility
+# This file is overwritten after every install/update
+# Persistent local customizations
+include qt6ct.local
+# Persistent global definitions
+ignore dns
```

Comment:
I just noticed this, why is it here?

There is no `dns` command in any profile.

Does this tool need internet access?